AfricanReligions/events/oldgods_ancestor_worship_events.txt

```diff
@@ -4571,6 +4832,295 @@ character_event = {
 	after = { clear_event_target = death_child }
 }
 
+#Yan Riga flavour event
+character_event = {
+	id = TOG.7530
```

Comment:
Let's create these in the AFRREL namespace, in their own file, so we don't risk incompatibility with the game.